voucher/src/main/java/com/ray/junho/voucher/domain/FixedAmountVoucher.java

```diff
@@ -0,0 +1,47 @@
+package com.ray.junho.voucher.domain;
+
+import java.util.Objects;
+
+public class FixedAmountVoucher implements Voucher {
+
+    private final long id;
+
+    private final int discountValue;
+
+    public FixedAmountVoucher(long id, int discountValue) {
+        this.id = id;
+        this.discountValue = discountValue;
+        validateDiscountValue(discountValue);
```

Comment:
```suggestion
        validateDiscountValue(discountValue);
        this.id = id;
        this.discountValue = discountValue;
```
검증 후 통과하면 set을 진행하는게 좀 더 괜찮아보여요!